src/main/scala/com/databricks/labs/overwatch/pipeline/Silver.scala

```diff
@@ -301,6 +303,13 @@ class Silver(_workspace: Workspace, _database: Database, _config: Config)
     append(SilverTargets.notebookStatusTarget)
   )
 
+  lazy private[overwatch] val sqlHistoryModule = Module(2020, "Silver_SqlHistory", this, Array(1016))
+  lazy private val appendSqlHistoryProcess = ETLDefinition(
+    BronzeTargets.sqlHistorySnapshotTarget.asIncrementalDF(sqlHistoryModule, SilverTargets.sqlHistoryTarget.incrementalColumns),
+    Seq(sqlHistoryTransform()),
+    append(SilverTargets.sqlHistoryTarget)
```

Comment:
This silver target is a WriteMode.merge but you aren't selecting any overlapping data. What happens when a query starts during run 1 and finishes in run 2 -- does it just never get its closure? Please select 2 lag days on this input and validate your keys and uniqueness across those keys to enable overlapping merges and close open queries across OW runs.

Reply:
added additional lag of 2 days